sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/DTDL/DigitalTwins/BuildingTwin.json

```diff
@@ -1,6 +1,9 @@
 ﻿{
     "$metadata": {
-        "$model": "dtmi:com:samples:Building;1"
+        "$model": "dtmi:com:samples:Building;1",
+        "AverageTemperature": {
+            "sourceTime": "2022-05-31T19:01:32.000000Z"
```

Comment:
To be sure -- this isn't `$sourceTime`?

Reply:
It's only "sourceTime". The "$" prefix is only used at the root of the $metadata block currently.